mobile/assets/i18n/nl-NL.json

```
@@ -261,20 +296,32 @@
   "permission_onboarding_permission_limited": "Beperkte toestemming. Geef toestemming tot foto's en video's in Instellingen om Immich een back-up te laten maken van je galerij en deze te beheren.",
   "permission_onboarding_request": "Immich heeft toestemming nodig om je foto's en video's te bekijken.",
   "profile_drawer_app_logs": "Logboek",
-  "profile_drawer_client_server_up_to_date": "App en server zijn up-to-date",
-  "profile_drawer_documentation": "Documentation",
+  "profile_drawer_client_out_of_date_major": "Mobiele App is niet meer up-to-date. Werk bij naar de nieuwste hoofdversie.",
+  "profile_drawer_client_out_of_date_minor": "Mobiele app is niet meer up-to-date. Werk bij naar de nieuwste onderversie.",
+  "profile_drawer_client_server_up_to_date": "Client en Server zijn up-to-date",
+  "profile_drawer_documentation": "Documentatie",
   "profile_drawer_github": "GitHub",
+  "profile_drawer_server_out_of_date_major": "Server is niet meer up-to-date. Werk bij naar de nieuwste hoofdversie.",
+  "profile_drawer_server_out_of_date_minor": "Server is niet meer up-to-date. Werk bij naar de nieuwste onderversie.",
   "profile_drawer_settings": "Instellingen",
   "profile_drawer_sign_out": "Uitloggen",
-  "profile_drawer_trash": "Trash",
+  "profile_drawer_trash": "Vuilbak",
```

```suggestion
  "profile_drawer_trash": "Prullenbak",
```
Vuilbak is Flemish, I believe.

---

Yes indeed. I'm from belgium. Should there be a different file (nl-BE)?